src/main/java/svc/models/Court.java

```diff
@@ -0,0 +1,156 @@
+package svc.models;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+// https://vladmihalcea.com/2017/03/29/the-best-way-to-map-a-onetomany-association-with-jpa-and-hibernate/
+
+@Entity
+public class Court {
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name="court_id")
+	private Long id;
+	
+	@Column(name="court_name")
+	private String name;
+	private String phone;
+	private String website;
+	private String extension;
+	private String address;
+	private String paymentSystem;
+	private String city;
+	private String state;
+	
+	@Column(name="zip_code")
+	private String zipCode;
+	private Double latitude;
+	private Double longitude;
+	
+	@OneToMany(cascade = CascadeType.ALL, orphanRemoval = true)
```

Comment:
Has the cascading and orphanRemoval been tested thoroughly to ensure it's giving you what you want?

Reply:
I'm not sure what you mean.  I'm not sure there is a way to unit test this.  When I ran this using local data, it seemed to work as expected, but for sure could use more testing from others.